doc/neps/nep-0018-array-function-protocol.rst

```diff
@@ -187,27 +193,60 @@ of these may decide that, given the available inputs, they are unable to
 determine the correct result. How do we call the right one? If several
 are valid then which has precedence?
 
-The rules for dispatch with ``__array_function__`` match those for
-``__array_ufunc__`` (see
+For the most part, the rules for dispatch with ``__array_function__``
+match those for ``__array_ufunc__`` (see
 `NEP-13 <http://www.numpy.org/neps/nep-0013-ufunc-overrides.html>`_).
 In particular:
 
 -  NumPy will gather implementations of ``__array_function__`` from all
    specified inputs and call them in order: subclasses before
-   superclasses, and otherwise left to right. Note that in some edge cases,
-   this differs slightly from the
+   superclasses, and otherwise left to right. Note that in some edge cases
+   involving subclasses, this differs slightly from the
    `current behavior <https://bugs.python.org/issue30140>`_ of Python.
 -  Implementations of ``__array_function__`` indicate that they can
    handle the operation by returning any value other than
    ``NotImplemented``.
 -  If all ``__array_function__`` methods return ``NotImplemented``,
    NumPy will raise ``TypeError``.
 
+One deviation from the current behavior of ``__array_ufunc__`` is that NumPy
```

Comment:
Hmm, that sounds like something we should do in `__array_ufunc__` as well. See #11306.

Reply:
Agreed.